web/app.fluidity.money/app/routes/$network/dashboard/home.tsx

```diff
@@ -313,8 +313,8 @@ export default function Home() {
   );
 
   const { width } = useViewport();
-  const isTablet = width < 850;
-  const isMobile = width < 500;
+  const isTablet = width < 850 && width > 500;
```

Comment:
Keep the original isMobile and isTablet - if isTable is true, it also implies isMobile is true